build/RuntimeStore.targets

```diff
@@ -0,0 +1,199 @@
+<Project>
+  <Import Project="common.props" />
+
+  <PropertyGroup>
+    <_DependencyBuildDirectory>$(RepositoryRoot).deps\build\</_DependencyBuildDirectory>
+    <_BuildScriptsDirectory>$(MSBuildThisFileDirectory)tools\scripts\</_BuildScriptsDirectory>
+    <_WorkRoot>$(RepositoryRoot).w\</_WorkRoot>
+    <_RuntimeStoreWorkDirectory>$(_WorkRoot).rw\</_RuntimeStoreWorkDirectory>
+    <_RuntimeStoreOutputDirectory>$(_WorkRoot).ro\</_RuntimeStoreOutputDirectory>
+    <_ExistingStoreDirectory>$(_WorkRoot).es\</_ExistingStoreDirectory>
+    <_RuntimeStoreReferencePackageDirectory>$(_WorkRoot)bin\Release\</_RuntimeStoreReferencePackageDirectory>
+    <_TemplatesDirectory>$(MSBuildThisFileDirectory)tools\templates\</_TemplatesDirectory>
+    <_SrcDirectory>$(RepositoryRoot)src\</_SrcDirectory>
+    <_ExistingManifestsDirectory>$(_SrcDirectory)Microsoft.AspNetCore.All\build\</_ExistingManifestsDirectory>
+    <_ArtifactsZipDirectory>$(ArtifactsDir)zip\</_ArtifactsZipDirectory>
+    <_DepsOutputDirectory>$(ArtifactsDir)deps\</_DepsOutputDirectory>
+  </PropertyGroup>
+
+  <Target Name="BuildAllMetapackage" DependsOnTargets="ResolveRepoInfo">
```

Comment:
This doesn't feel like the right approach. Maybe we can run the repo task as part of building the package instead of having this separate target. All that's needed is to populate the references before building.